cli/pcluster/configure/utils.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2013-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the 'LICENSE.txt' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+# fmt: off
+import functools
+import logging
+import sys
+from builtins import input
+
+import boto3
+from botocore.exceptions import BotoCoreError, ClientError
+
+LOGGER = logging.getLogger("pcluster.pcluster")
```

Comment:
what's the best practice? put in in utils and then import it from there?

Reply:
I was referring more to the string that is passed to getLogger.

From https://docs.python.org/3/library/logging.html:

>The logger name hierarchy is analogous to the Python package hierarchy, and identical to it if you organise your loggers on a per-module basis using the recommended construction logging.getLogger(__name__). That’s because in a module, __name__ is the module’s name in the Python package namespace.